### PR TITLE
feat(deno): update to 1.25.0

### PR DESCRIPTION
**Pull request by SeparateRecords**
Can't update automatically because there's a fair amount that has to be cleaned up manually in this spec, but _wow_ it is much easier to scan the Fig spec diff than diff the deno CLI's source code!! Using the generated spec helped so much.

### CODE REVIEW

**Comment by fedeci** [comment]
Can we use diff-versioned specs for deno?

File: src/deno.ts [hunk 1]
```diff
@@ -9,7 +8,7 @@ import {
 
 const completion: Fig.Spec = {
   name: "deno",
-  description: "A secure JavaScript and TypeScript runtime",
+  description: "A modern JavaScript and TypeScript runtime",
```
Comment (fedeci) [inline src/deno.ts:11]: Isn't it secure anymore? 😆

**Review by fedeci — COMMENTED** [review]
I imagine this is not yet ready to be merged, ping me when it is!

File: src/deno.ts (hunk 1 above)
Comment (SeparateRecords) [reply src/deno.ts:11]: I know!! I thought the same thing when I saw their tagline had changed 😄

**Comment by SeparateRecords** [comment]
> Can we use diff-versioned specs for deno?

Possibly! I just used `deno completions fig` to generate the updated one then merged them. Not entirely sure how to create a diff-versioned spec without using the github action though - could be missing something but couldn't find it in the docs. Could you point me in the right direction?

**Comment by SeparateRecords** [comment]
Also this is ready to be merged rn @fedeci

**Comment by fedeci** [comment]
> > Can we use diff-versioned specs for deno?
> 
> 
> 
> Possibly! I just used `deno completions fig` to generate the updated one then merged them. Not entirely sure how to create a diff-versioned spec without using the github action though - could be missing something but couldn't find it in the docs. Could you point me in the right direction?

https://fig.io/docs/concepts/advanced-specs that's it!

**Comment by SeparateRecords** [comment]
🤦 right in front of me... thank you!!

File: src/deno.ts [hunk 2]
```diff
@@ -287,7 +280,6 @@ const completion: Fig.Spec = {
           name: "files",
           isVariadic: true,
           isOptional: true,
-          template: "filepaths",
```
Comment (fedeci) [inline src/deno.ts:290]: Is this intended, then?

Comment (SeparateRecords) [reply src/deno.ts:290]: Not intended, thanks for catching this

**Pull request merged**